rigex-front-end/src/pages/Login/Login.js

```diff
@@ -1,7 +1,159 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { gql, useLazyQuery } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+import {
+  Input,
+  Button,
+  IconButton,
+  Divider,
+  FormControl,
+  InputLabel,
+  Grid,
+} from '@material-ui/core';
+import {
+  Mail,
+  Lock,
+  VisibilityOutlined,
+  VisibilityOffOutlined,
+} from '@material-ui/icons';
+
+import { loginActions } from '../../store/login/action';
+import { LOGIN_USER } from '../../constants/serviceAPI';
+import history from '../../utils/history';
+import routePaths from '../../constants/routePaths';
+import logo from '../../assets/large-logo.png';
+import './login.scss';
 
 const Login = () => {
-  return <h1>Login Works!</h1>;
+  const [userState, setUserState] = useState({ email: '', password: '' });
+  const [errorMessage, setErrorMessage] = useState('');
+  const [passwordIsVisible, setPasswordIsVisible] = useState(false);
+  const [login, { loading, error, data }] = useLazyQuery(LOGIN_USER);
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    login({
+      variables: { userInput: { ...userState } },
+    });
+  };
+
+  useEffect(() => {
+    if (error) {
+      setErrorMessage(error);
+    }
+
+    if (data) {
+      dispatch(loginActions.setLoggedInUser(data.login));
+    }
+  }, [loading, error, data]);
```

Comment:
A warning is shown on this line that says `React Hook useEffect has a missing dependency: 'dispatch'. Either include it or remove the dependency array`. Have you tried adding dispatch to the list of dependencies?